src/Humanizer/DateToOrdinalWordsExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Globalization;
+using Humanizer.Configuration;
+
+namespace Humanizer
+{
+    /// <summary>
+    /// Humanizes DateTime into human readable sentence
+    /// </summary>
+    public static class DateToOrdinalWordsExtensions
+    {
+        /// <summary>
+        /// Turns the provided date into ordinal words
+        /// </summary>
+        /// <param name="input">The date to be made into ordinal words</param>
+        /// <returns>The date in ordinal words</returns>
+        public static string ToOrdinalWords(this DateTime input)
+        {
+            return input.Day.Ordinalize() + input.ToString(" MMMM yyyy");
```

Comment:
Good point, there are plenty of countries that do MMDDYY and plenty that do YYMMDD. I'm working on an update to be localised. Can I ask if I just put up a PR request with only a couple of languages is that ok? Others could then add more in other languages?